lib/active_record/associated_object/object_association.rb

```diff
@@ -1,4 +1,6 @@
 module ActiveRecord::AssociatedObject::ObjectAssociation
+  extend ActiveSupport::Concern
```

Comment:
Pet-peeve of mine that I've been starting to avoid `ActiveSupport::Concern`s, so let's do:

```suggestion
  def self.included(klass) = klass.extend ClassMethods
```